Tests/L0/Maven/_suite.ts

```diff
@@ -706,14 +705,13 @@ describe('maven Suite', function() {
         var testSourceDirectory = path.join(__dirname, 'data');
 
         // Act
-        var exampleResult = pmd.processPmdOutput(testSourceDirectory);
+        var exampleResult = pmd.collectPmdOutput(path.join(testSourceDirectory, 'singlemodule'));
```

Comment:
You don't seem to use testSourceDirectory anywhere else, so you could combine the 2 calls to path.join into a single one